apps/suite-launch/test-presets/golden-path-v01-11-2023/firelens-datajet.json

```diff
@@ -0,0 +1,104 @@
+{
+  "component": "synchronizer",
+  "config": {
+    "repeat": 100,
+    "waitBefore": 5,
+    "waitAfter": 10,
+    "waitBetween": 0,
+    "isAsync": true
+  },
+  "children": [
+    {
+      "comment": "FireLens logs to standard out (move forward)",
+      "generator": {
+        "name": "basic",
+        "config": {
+          "contentLength": 100,
+          "batchSize": 10,
+          "key": "log",
+          "addNewline": true
+        }
+      },
+      "datajet": {
+        "name": "forward",
+        "config": {
+          "host": "{{{endpoint_flb}}}",
+          "port": 6270,
+          "tagPrefix": "Application-firelens-"
+        }
+      },
+      "stage": {
+        "batchRate": {{{kbps_throughput_forward}}},
+        "maxBatches": 100000000000
```

Comment:
Does this mean the datajet will stop at some point?

Reply:
Yes, but in our case we only test with 10000 as maximum, so that's 115 days, which should be long enough. It may be good to get rid of this option actually...